README.md

```diff
@@ -159,18 +160,18 @@ Before starting this codelab, make sure that you've installed:
                                         localName, remoteName) {
         const candidatesCollection = roomRef.collection(localName);
 
-        peerConnection.addEventListener('icecandidate', event -> {
+        peerConnection.addEventListener('icecandidate', event => {
```

Comment:
ah, thanks.  I did get it right in my version of the solution branch, but being primarily java-based myself, this doesn't set my alarms off as readily.  Dunno why they chose skinny arrows for java... *grumble*